### PR TITLE
highlight upgrades which have soaked enough

### PR DESCRIPTION
**Pull request by patjlm**

### CODE REVIEW

File: tools/qontract_cli.py [hunk 1]
```diff
@@ -253,6 +253,15 @@ def cluster_upgrade_policies(ctx, cluster=None, workload=None,
     results = []
     upgrades_cache = {}
 
+    def soaking_str(soaking, soakdays):
+        sorted_soaking = sorted(soaking.items(), key=lambda x: x[1])
+        if ctx.obj['options']['output'] == 'md':
+            for i, data in enumerate(sorted_soaking):
+                v, s = data
+                if s > soakdays:
+                    sorted_soaking[i] = (v, f'{s} :tada:')
```
Comment (maorfr) [inline tools/qontract_cli.py:262]: :heart:

**Review by maorfr — APPROVED** [review]
LGTM

how do we tell people that tada means ready for upgrade?

**Comment by patjlm** [comment]
> how do we tell people that tada means ready for upgrade?

I am debating with myself if the `md` output could include some text above the table to provide some explanations

**Comment by patjlm** [comment]
> > how do we tell people that tada means ready for upgrade?
> 
> I am debating with myself if the `md` output could include some text above the table to provide some explanations

I added an explanation of the various columns in the `md` output

**Pull request merged**